README.md

```diff
@@ -21,6 +21,25 @@ ranging from major cloud providers to popular SaaS companies.  Our end goal is
 to offer this specification to the
 [Cloud Native Computing Foundation](https://www.cncf.io/).
 
+## Working Group process
+
+The CNCF Serverless WG is working to formalize the [specification](spec.md)
+based on [design goals](spec.md#design-goals) which focus on interoperability
+between systems which generate and respond to events.
+
+In order to achieve these goals, the Serverless WG must describe:
+- Common attributes of an *event* that facilitate interoperability
+- One or more common architectures that are in active use today or planned to be
+built by WG members
+- How events are transported from source to action via at least one protocol
+- Authorization model(s) that control access and execution permissions
```

Comment:
I think its important to talk about having implementations of the spec prior to going to v1.0.
No code, no verification, no spec :-)

Reply:
@ultrasaurus not sure if the goal is to define the common attributes of an event (there can be many types of events w/o common structure), rather the attributes that allow us to pass the event from the source to the function reliably, observe its movement, and decipher its content (i.e. what i call the event envelop). The interop at the event level will be achieved by (future) definition of some common event types/schema.

Reply:
The [roadmap](https://github.com/cloudevents/spec/blob/master/roadmap.md) calls for creating reference implementations by v0.2, I assume with the expectation that implementations will be updated as we refine the spec. 

I'm resistant to including implementations here because I see this section as being the sticking points around interoperability that the spec should address, while implementations are a demonstration of the spec working.

How strongly do you feel that the implementations should be included here?

Reply:
re: `Authorization model(s)...`  can someone elaborate on this one? access and permissions for what?

Reply:
@duglin LGTM

Reply:
@duglin, An example use case for handling Authorization is that an Event Consumer receives an Event, and will take an action of, say, unlocking the door. First it wants to confirm that this user has permission to unlock this door.

Reply:
@rachelmyers yup - I agree with that kind of action/check being done - what's not clear to me is whether that's part of what our spec needs to define. That feels out of scope w.r.t. "defining an event" - and gets more info the processing model of the event.